catapult/release.py

```diff
@@ -132,6 +132,9 @@ def _get_versions(client, bucket, key):
         if version["Key"] != key:
             continue
 
+        if version["VersionId"] is None or version["VersionId"] == "null":
+            continue
+
```

Comment:
This is the (maybe) bug fix

Reply:
What's the bug?

Reply:
There's a failing test with the s3 bucket versioning. The test expects we skip an object without a version, so this adds that explicitly.

`test_get_releases_skips_non_versioned_objects` puts a file in s3, then turns on versioning, and adds another file, then asserts that only the second file is returned from `get_releases`.